scripts/upload.gradle

```diff
@@ -128,24 +130,20 @@ afterEvaluate { project ->
         }
 
         task androidJavadocsJar(type: Jar, dependsOn: androidJavadocs) {
+            classifier = "javadoc"
             from androidJavadocs.destinationDir
             baseName "${JAR_PREFIX}${project.name}${JAR_POSTFIX}"
         }
 
         task androidSourcesJar(type: Jar) {
+            classifier = 'sources'
             from project.android.sourceSets.main.java.source
             baseName "${JAR_PREFIX}${project.name}${JAR_POSTFIX}"
         }
-    }
 
-    artifacts {
-        if (project.getPlugins().hasPlugin('android') || project.getPlugins().hasPlugin('android-library')) {
-            archives androidSourcesJar {
-                classifier "sources"
-            }
-            archives androidJavadocsJar {
-                classifier "javadoc"
-            }
+        artifacts {
```

Comment:
The indentation change is because I merged the two same if statements into one (checking whether the project has the android application or library plugin applied).

@TWiStErRob yup that's right. You can also see that [here](https://github.com/chrisbanes/gradle-mvn-push/blob/master/gradle-mvn-push.gradle)